galasa-parent/dev.galasa.framework.api.authentication/src/main/java/dev/galasa/framework/api/authentication/internal/routes/AuthTokensRoute.java

```diff
@@ -233,4 +279,31 @@ private void addTokenToAuthStore(String clientId, String jwt, String description
         }
         logger.info("Stored token record in the auth store OK");
     }
+
+    private List<AuthToken> convertAuthStoreTokenIntoTokenBean(List<IInternalAuthToken> authStoreTokens){
+
+        List<AuthToken> tokensToReturn = new ArrayList<>();
+        
+        for (IInternalAuthToken token : authStoreTokens) {
+
+            User user = new User(token.getOwner().getLoginId());
+            tokensToReturn.add(new AuthToken(
+                    token.getTokenId(),
+                    token.getDescription(),
+                    token.getCreationTime(),
+                    user));
+        }
+
+        return tokensToReturn;
+
+    }
+
+    private void validateLoginId(String loginId, String servletPath) throws InternalServletException {
+
+        if (loginId ==null || loginId.trim() == null) {
+            ServletError error = new ServletError(GAL5400_BAD_REQUEST, servletPath);
```

Comment:
Would be good to have a more specific error message to indicate that the loginId was provided but was badly formed instead of a generic bad request message